src/MarketingLayout/test/MarketingLayout.tsx

```diff
@@ -21,6 +22,7 @@ function marketingLayoutWithAllProps() {
       actions={<div />}
       title="title"
       description="description"
+      badge={<Badge size="small">badge</Badge>}
```

Comment:
Missing hiddenBadge prop

```suggestion
      badge={<Badge size="small">badge</Badge>}
      hiddenBadge
```